unit_tests/globalaccess.h

```diff
@@ -23,4 +23,6 @@
 #define PASS_ENGINE_PARAMETER_SUFFIX , PASS_ENGINE_PARAMETER_SIGNATURE
 
 #define CONFIG(x) engineConfiguration->x
+#define CONFIG_POINTERS(x) &CONFIG(x), &activeConfiguration.x
```

Comment:
are these used?

Reply:
nope. thought I had deleted that but must have missed it.